app/src/main/java/com/example/dine_and_donate/Activities/HomeActivity.java

```diff
@@ -123,4 +150,30 @@ private void navigationHelper(Class navigateToClass) {
         intent.putExtra(User.class.getSimpleName(), Parcels.wrap(mCurrentUser));
         startActivity(intent);
     }
+
+    private void setUpNotificationWorker() {
+        Calendar calendar = Calendar.getInstance();
+        // 8.00 (8 AM)
+        calendar.set(Calendar.HOUR_OF_DAY, 12);
+        calendar.set(Calendar.MINUTE, 0);
+        calendar.set(Calendar.SECOND, 0);
+
+        Intent myIntent = new Intent(HomeActivity.this, MyReceiver.class);
+        pendingIntentam = PendingIntent.getBroadcast(HomeActivity.this, 0,myIntent,0);
```

Comment:
small nit: put spaces after the commas like "HomeActivity.this, 0, myIntent, 0);"